src/apis/clubhome.js

```diff
@@ -0,0 +1,56 @@
+import instance from "apis";
+
+// 동아리 정보 불러오기
+export const getClubInfo = async () => {
+  return await instance.get(`api/club/home/1`);
+};
```

Comment:
```suggestion
export const getClubInfo = () => {
  return instance.get(`api/club/home/1`);
};
```
여기 있는 함수 전부 이렇게 해도 될거 같아요